src/config/schema-transformer/to_bgp.py

```diff
@@ -791,7 +799,13 @@ def add_route(self, prefix, next_hop):
                 route_target=[self.get_route_target()])
             static_route_entries.add_route(static_route)
         left_ri.obj.set_static_route_entries(static_route_entries)
-        _vnc_lib.routing_instance_update(left_ri.obj)
+        try:
+            _vnc_lib.routing_instance_update(left_ri.obj)
+        except NoIdError:
+            _sandesh._logger.error(
+                "Cannot add route. Id not found for %s", left_ri.name)
+            return
```

Comment:
If you return from here, update_route_target_list is not done, leaving ri incomplete. Should we move below call 'update_route_target_list' before this try catch ?